lib/tasks/test.rb

```diff
@@ -18,6 +18,7 @@
   task :setup do
     testing.each_pair do |app, _task|
       chdir PROJ_DIR.join(app.to_s) do
+        sh "yarn install" if app.to_s == "apps/dashboard"
```

Comment:
Rather than require this condition be updated as new apps use yarn, should execute this yarn command if some file specific to yarn exists like the lock file.

Reply:
I wonder here if we should instead be doing `RAILS_ENV=test bin/setup` or `RAILS_ENV=development bin/setup` instead of explicitly calling `sh "bundle install --with development test"` and thus wanting to also above call `         sh "yarn install" if app.to_s == "apps/dashboard"`...

Reply:
Resolved in ba80ac5